client/src/components/UserProfile.js

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+const UserProfile = ({ user }) => {
+  return (
+    <div>
+      <h1>This is a page built specifically for a user to see</h1>
+      <h2></h2>
+    </div>
+  )
+}
```

Comment:
We expect complete features to have the data that we are expecting to see and not filler text. We should at the very least see some information about the user, even if in the future we plan on having something completely different on this page.